modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,286 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="aws-load-balancer-operator-install_{context}"]
+= Installing an AWS Load Balancer Operator
+Users that wish to install an AWS Load Balancer Operator (ALBO) can do so as long as the following prerequisites are met:
+
+.Prerequisites
+
+* An existing
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+{product-title} (ROSA)
+endif::openshift-rosa[]
+cluster with bring-your-own-VPC (BYO-VPC) configuration across multiple availability zones (AZ) installed in STS mode
+* Cluster-admin access
+* Access to modify the VPC and subnets of the created
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster
+// * AWS account credentials
+* Configured
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+CLI (`rosa`)
+* Configured Amazon Web Services (AWS) CLI
+* Configured OpenShift (oc) CLI
+* OpenShift Container Platform (OCP) 4.13 or later
+
+[NOTE]
+====
+When installing an ALBO for use with
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+a ROSA
+endif::openshift-rosa[]
+cluster in an AWS Local Zone (LZ), the AWS LZ must be enabled for the account and AWS Elastic Load Balancing v2 (ELBv2) services must be available in the AWS LZ.
+====
+.Procedure
+
+. Identify the cluster infrastructure ID and the cluster OpenID Connect (OIDC) DNS by running the following commands:
+.. Identify the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster INFRA ID:
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name> | grep -i 'Infra ID'
+----
++
+or
++
+[source,terminal]
+----
+$ oc get infrastructure cluster -o json | jq -r '.status.infrastructureName'
+----
+
+.. Identify the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster OIDC DNS:
++
+--
+[source, terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name> | grep -i 'OIDC'
+----
+
+Save the output from both commands. You will use this information in future steps within this procedure.
+
+--
+. Create and manage AWS Identity and Access Management (IAM) roles and policies:
++
+.. Create the AWS IAM policy required for the ALBO, which can be found link:https://github.com/openshift/aws-load-balancer-operator/blob/61b55feef26e8dfd9d1b2a3300f80ba572f98d58/docs/prerequisites.md#option-2-using-aws-cli[here].
++
+.. Create the AWS IAM policy required for the AWS Load Balancer Controller (ALBC), which can be found link:https://github.com/openshift/aws-load-balancer-operator/blob/61b55feef26e8dfd9d1b2a3300f80ba572f98d58/docs/install.md#option-2-using-the-aws-cli[here].
```

Comment:
```suggestion
.. Create the AWS IAM policy required for the AWS Load Balancer Controller (ALBC), as explained in link:https://github.com/openshift/aws-load-balancer-operator/blob/61b55feef26e8dfd9d1b2a3300f80ba572f98d58/docs/install.md#option-2-using-the-aws-cli[the ALBC installation documentation].
```
- Per [the ISG](https://www.ibm.com/docs/en/ibm-style?topic=elements-web-ip-email-addresses#referring-to-web-addresses), when you refer to a web page use a meaninful descriptive label.

Reply:
Done